I'm sorry, but an uncaught exception occurred.

While running game code:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Ash\\AppData\\Roaming/RenPy/Alphademo-1686417027\\persistent.1714567857.tmp'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\Ash\Downloads\Mingid äpid\renpy-8.1.3-sdk\renpy\bootstrap.py", line 275, in bootstrap
    renpy.main.main()
  File "C:\Users\Ash\Downloads\Mingid äpid\renpy-8.1.3-sdk\renpy\main.py", line 684, in main
    renpy.persistent.update(True)
  File "C:\Users\Ash\Downloads\Mingid äpid\renpy-8.1.3-sdk\renpy\persistent.py", line 401, in update
    save()
  File "C:\Users\Ash\Downloads\Mingid äpid\renpy-8.1.3-sdk\renpy\persistent.py", line 419, in save
    renpy.loadsave.location.save_persistent(compressed)
  File "C:\Users\Ash\Downloads\Mingid äpid\renpy-8.1.3-sdk\renpy\savelocation.py", line 545, in save_persistent
    l.save_persistent(data)
  File "C:\Users\Ash\Downloads\Mingid äpid\renpy-8.1.3-sdk\renpy\savelocation.py", line 374, in save_persistent
    with open(fn_tmp, "wb") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Ash\\AppData\\Roaming/RenPy/Alphademo-1686417027\\persistent.1714567857.tmp'

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
Mull aja meres Ver.1.0
Wed May  1 15:51:31 2024
